arrays_and_loops/arrays_and_loop.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"fmt"
+	"math"
+)
+
+func main() {
+	var arr [5]int
+	for i:=0;i<len(arr);i++ {
```

Comment:
This isn't the best way to use `for` statement in golang. probably the most golanguintisc way is this one:
```suggestion
	for i := range arr {
```

Can you update the code to use the suggested form?

Reply:
for sure, you're right why define a loop to iterate over an array if i have 'range' func used for